.changeset/tasty-cougars-dress.md

```diff
@@ -0,0 +1,6 @@
+---
+"@redocly/openapi-core": patch
+"@redocly/cli": patch
+---
+
+Updated `operation-tag-defined` configurable rule.
```

Comment:
```suggestion
Updated the `operation-tag-defined` built-in rule to check that the globally defined tags are used at the operation level.
```

Reply:
I was only able to find the built-in rule `operation-tag-defined`. I also think it would be helpful to add some information to the rule's documentation with this update - https://redocly.com/docs/cli/rules/oas/operation-tag-defined